src/dots.jsx

```diff
@@ -25,8 +25,9 @@ export var Dots = React.createClass({
       slidesToScroll: this.props.slidesToScroll
     });
 
-    var dots = Array.apply(null, {length: dotCount}).map((x, i) => {
+    var dots = [];
 
+    for (var i=0; i<dotCount; i++) {
```

Comment:
@sorrycc I don't see a need for this change. Please revert changes on this particular file. 

I'll look at this PR in the weekend. I am bit busy with my college work.